### ISSUE TITLE
Searcning for musicians/developpers to do a new kind of music using mido

### ISSUE THREAD

**Issue by TheUnknownScientist** [mido/mido#253]
Dear All,

I am using mido to generate unique interpretations of a given music sheet with a computer. I am searching for developers/musicians to help me with this endeavour. Check out this [git depo](https://framagit.org/APianoInterpreter/furambre/) to further understand what I mean and generate your own interpretation of Fur Elise by Beethoven.

Best, 
TheUnknownScientist

P.S: I use this git depo issue as a kind of forum. Sorry for that but I could not find a working forum.

**Comment by olemb** [comment]
Cool project! I think it can really benefit from the timing improvements that are in the pipeline (see request #161).

No problem posting here under the circumstances. Due to the lack of a forum I've decided to try out GitHub's new discussion feature and have set up [a discussion page](https://github.com/mido/mido/discussions). I just posted a welcome message to start things off.

Feel free to post over there!
